core/cloudflow-operator/src/main/scala/cloudflow/operator/CloudflowApplication.scala

```diff
@@ -186,42 +234,76 @@ object CloudflowApplication {
   }
 
   object PodStatus {
+    val Pending          = "Pending"
+    val Running          = "Running"
+    val Terminating      = "Terminating"
+    val Terminated       = "Terminated"
+    val Succeeded        = "Succeeded"
+    val Failed           = "Failed"
+    val Unknown          = "Unknown"
+    val CrashLoopBackOff = "CrashLoopBackOff"
+
+    val ReadyTrue  = "True"
+    val ReadyFalse = "False"
```

Comment:
couldn't we have done with a single `val Ready = "False"` or something booleanish ?

Reply:
These are constants used to set ready to true or false as a string value. (terrible, but this is how it was done, so need to keep that around)